support-frontend/assets/pages/digital-subscription-checkout/components/digitalCheckoutForm.test.tsx

```diff
@@ -38,10 +39,20 @@ function setUpStore(initialState: WithDeliveryCheckoutState) {
 	return store;
 }
 
+jest.mock('helpers/globalsAndSwitches/globals', () => ({
+	__esModule: true,
+	isSwitchOn: jest.fn(),
+	getSettings: jest.fn(),
+	getGlobal: jest.fn(),
+}));
+
+const mock = (mockFn: unknown) => mockFn as jest.Mock;
+
 describe('Digital checkout form', () => {
 	// Suppress warnings related to our version of Redux and improper JSX
```

Comment:
Not related to your PR but I wonder if this comment and suppression is still needed?

Reply:
Good point, will check this out now!

Reply:
We still need to suppress a few warnings here related to improper JSX but I'll update the comment to remove reference to Redux